### ISSUE TITLE
Training code for the 1st stage

### ISSUE THREAD

**Issue by Guo-Stone** [Vecteur-libre/EnzyControl#2]
Dear authors, you have mentioned in your paper that you apply two-stage training. Could you please tell me the loss and the code for the 1st training stage?
